OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,433 @@
+Building an OUD image with WebLogic Image Tool
+===========================================
+
+## Contents
+
+1. [Introduction](#1-introduction)
+2. [Prerequisites](#2-prerequisites)
+3. [Setup WebLogic Image Tool](#3-setup-weblogic-image-tool)
+4. [Download the required packages/installers](#4-download-the-required-packagesinstallers-patches)
+5. [Required build files](#5-required-build-files)
+6. [Additional build commands](#6-additional-build-commands)
+7. [Steps to Create OUD image](#7-steps-to-create-oud-image)
+8. [Sample Dockerfile generated with imagetool](#8-sample-dockerfile-generated-with-imagetool)
+
+# 1. Introduction
+
+This README describes the steps involved in building an OUD image with the WebLogic Image Tool.
+
+# 2. Prerequisites
+
+The following prerequisites are necessary before building OUD Docker images with Image Tool:
+
+* A working installation of Docker 18.03.1 or later
+* Bash version 4.0 or later, to enable the <tab> command complete feature.
+* JAVA_HOME environment variable set to the location of your JDK e.g:  /scratch/export/oracle/product/jdk
+
+# 3. Setup WebLogic Image Tool
+
+* Download the latest WebLogic Image Tool version from the release [page](https://github.com/oracle/weblogic-image-tool/releases).
+* Unzip the release ZIP file to a desired \<work directory\>.
```

Comment:
I recommend using the `tar.gz` archive as that's more likely to be installed on a Linux server.